Source/EasyNetQ/LegacyRpcConventions.cs

```diff
@@ -0,0 +1,13 @@
+﻿using EasyNetQ.Topology;
+
+namespace EasyNetQ
+{
+    public class LegacyRpcConventions : Conventions
+    {
+        public LegacyRpcConventions(ITypeNameSerializer typeNameSerializer)
+            : base(typeNameSerializer)
+        {
+            RpcResponseExchangeNamingConvention = (type) => Exchange.GetDefault().Name;
```

Comment:
It seems parentheses are redundant here.

Reply:
yes they are because [I took from here](https://github.com/EasyNetQ/EasyNetQ/blob/develop/Source/EasyNetQ/Conventions.cs#L75) 😄  and probably I should fix both, thank you.